templates/layouts/bootstrap.html.ep

```diff
@@ -132,6 +132,9 @@
           %= content
       </div>
 
+    % my $fullscreen = $self->param('fullscreen');
+
+    % if (!$fullscreen) {
```

Comment:
nope. If you need a different layout for a specific template, specify that in the template